product_company_id/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+{
+    "name": "Product Company ID",
```

Comment:
why not product_company only? I mean, why not removing "id" everywhere?
Not a blocker!

Reply:
applied